### PR TITLE
Add TLS (HTTPS) section to gke.md

### PR DESCRIPTION
**Pull request by tamanyan**
**What this PR does / why we need it**:

Update gke.md in order to add the instruction of how to setup TLS (HTTPS) on GKE by using [Let's Encrypt](https://letsencrypt.org/).

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

Nothing.

**Special notes for your reviewer**:

Nothing.

### CODE REVIEW

**Comment by rainest** [comment]
Although it's arguably covered in the gke-letsencrypt steps, I'd recommend noting that the Ingress creation at the end is effectively step 4 of their instructions, and that steps 5 & 6 should be completed afterward.

**Comment by tamanyan** [comment]
@rainest I added the timing of the Ingress creation

**Comment by hbagdi** [comment]
ping @rainest

File: deploy/gke.md [hunk 1]
```diff
@@ -216,13 +216,13 @@ spec:
 
 You need to set your API with HTTPS in order to expose your service securely. In this section, I will explain how to secure it with [Let’s Encrypt](https://letsencrypt.org/).
 
-1. Register your domain
+##### 1. Register fyour domain
```
Comment (coopr) [inline deploy/gke.md]: fyour -> your

**Comment by hbagdi** [comment]
@tamanyan Merged manually. Thank you for the contribution!

**Pull request closed without merging**